### ISSUE TITLE
Spin prior documentation

### ISSUE THREAD

**Issue by angelgarron** [hemantaph/ler#7]
Hi @hemantaph,

There is a table in the documentation https://ler.readthedocs.io/en/latest/GW_events.html#Sampling-Methods-for-Compact-Binary-Sources where the parameters sampled are listed, but the spin-related parameters are missing.

Also, in https://ler.readthedocs.io/en/latest/autoapi/ler/gw_source_population/cbc_source_parameter_distribution/index.html#ler.gw_source_population.cbc_source_parameter_distribution.CBCSourceParameterDistribution, the description for the `spin_precession` argument:

```
If spin_zero=True and spin_precession=True, spin parameters are sampled for precessing binaries. if spin_zero=True and spin_precession=False, spin parameters are sampled for aligned/anti-aligned spin binaries. default: False
```

should say `If spin_zero=False and spin_precession=True, spin parameters are sampled for precessing binaries. if spin_zero=False and spin_precession=False,...`

CC @dbkeitel

**Comment by hemantaph** [comment]
Thank you for pointing it out. I have made the following changes

1. I have included information on the spin priors in the [documentation](https://ler.readthedocs.io/en/latest/GW_events.html#Sampling-Methods-for-Compact-Binary-Sources)
2. I have fixed the typo error to "spin_zero=False and spin_precession=True" in the [code](https://ler.readthedocs.io/en/latest/autoapi/ler/gw_source_population/cbc_source_parameter_distribution/index.html#ler.gw_source_population.cbc_source_parameter_distribution.CBCSourceParameterDistribution)

**Comment by dbkeitel** [comment]
Thanks  @hemantaph ! I think you've missed the second sentence of the second place though, which is currently `if spin_zero=True and spin_precession=False` and I think it should be
`If spin_zero=False and spin_precession=False`.

**Comment by hemantaph** [comment]
You are right. I just made the changes, it will be reflected after in the documentation after a while.
